src/modules/NtpClient.rb

```diff
@@ -409,6 +409,16 @@ def Import(settings)
       true
     end
 
+    # Merges config to existing system configuration. It is useful for delayed write.
+    # When it at first set values, then chrony is installed and then it writes. So
```

Comment:
Hm, I do not really understand this.

Reply:
I try to explain here that if you set values, it works on parsed configuration file, but if package is not yet installed, then it create new configuration file which is missing many stuff. So if package is installed later, then this method re-apply changes on top of now parsed file.